### PR TITLE
F2 shortcut 

### PR DESCRIPTION
**Pull request by petrican**

### CODE REVIEW

**Comment by petrican** [comment]
https://www.loom.com/share/fd3bbcfc541a4601904446457bc1aa91

File: frontend/src/components/table/TableItem.js [hunk 1]
```diff
@@ -188,6 +188,10 @@ class TableItem extends PureComponent {
           this.setState({ valueBeforeEditing: fieldsByName[property].value });
         const inp = String.fromCharCode(e.keyCode);
         if (/[a-zA-Z0-9]/.test(inp) && !e.ctrlKey && !e.altKey) {
+          if (e.keyCode === 113) {
```
Comment (siemiatj) [inline frontend/src/components/table/TableItem.js:191]: Can we just store it in constants ?

**Review by siemiatj — APPROVED** [review]

**Comment by siemiatj** [comment]
Just one minor thing. Other than that, LGTM

File: frontend/src/components/table/TableItem.js (hunk 1 above)
Comment (petrican) [reply frontend/src/components/table/TableItem.js]: Done. Thanks

**Pull request merged**